offices/students-2/index.html

```diff
@@ -1,11 +1,10 @@
 
 {% set query = queries.office %}
-{% set results = query.search_with_url_arguments(q="_id:students-2",size=1) %}
-{% set office = results | first %}
+{% set office = get_document("office", "students-2") %}
 {% set path = '/offices/' + office.slug + '/' %}
 {% set resource_title = "Student resources" %}
 {% set activity_type = "Students" %}
 
-{% set initiatives = queries.initiative.search_with_url_arguments(q="related_office:students-2") %}
+{% set initiatives = queries.initiative.search_with_url_arguments(filter_related_office="students-2") %}
```

Comment:
Yep, @willbarton is right.  You can filter for anything where you match a term, a boolean, etc with filter_[field], and anything with a range (dates, numbers) with filter_range_[field]_[operator]. (operator being gt/lt/gte/lte)

To inspect the fields for different types, I highly recommend installing elasticsearch-head if @himedlooff didn't already force it upon you:

(if using homebrew) `[your-homebrew-dir]/Cellar/elasticsearch/[your-homebrew-version]/bin/plugin -install mobz/elasticsearch-head`

(otherwise) `elasticsearch/bin/plugin -install mobz/elasticsearch-head`

Then go to http://localhost:9200/_plugin/head/